src/main/webapp/app/exam/participate/exercises/exam-exercise-update-highlighter/exam-exercise-update-highlighter.component.ts

```diff
@@ -98,23 +147,25 @@ export class ExamExerciseUpdateHighlighterComponent implements OnInit {
      * @param diffs Array of diff tuples. (from DiffMatchPatch)
      * @return the HTML representation as string with markdown intact.
      */
-    diffPrettyHtml = function (diffs: Diff[]): string {
+    private diffPrettyHtml(diffs: Diff[]): string {
         const html: any[] = [];
+        const colorForAddedText = this.isDark ? '#006400' : '#e6ffe6';
+        const colorForDeletedText = this.isDark ? '#a22' : '#ffe6e6';
```

Comment:
Do these hardcoded colors really follow our theming guidelines?
Can we use css classes and variables in our scss files instead?

Reply:
I guess, we can, I just adopted what was already there but I try to refactor it.